src/appshell/qml/DevTools/KeyNav/KeyNavSubSection.qml

```diff
@@ -34,8 +34,21 @@ Rectangle {
     height: 40
     width: btns.childrenRect.width
 
-    border.color: "#75507b"
-    border.width: keynavsec.active ? 4 : 0
+    Rectangle {
```

Comment:
I believe you need to create a separate component (NavigatioBorder) and reuse it instead of copy-pasting this code in all components

Reply:
Understood!

Reply:
Fixed :)